R/loadReg2.R

```diff
@@ -444,14 +463,17 @@ predictSolute.loadReg2 <- function(
       }
     }
   }
-  
   # Add dates if requested
   if(date) {
     if(!is.data.frame(preds)) {
       preds <- data.frame(fit=preds)
     }
     # prepend the date column
-    preds <- data.frame(date=getCol(metadata, newdata, "date"), preds)
+    #preds <- data.frame(date=getCol(metadata, newdata, "date"), preds)
+    #predLoad returns the dates, don't need to get them from metadata
```

Comment:
ok, i've looked and still think the date column needs to be added - always, in fact. `preds` is not the direct output from `predLoad` or `predConc`. it's either a vector (https://github.com/wdwatkins/loadflex/blob/9e4e520e0b88a04ce594aa1e7b5048e26dae0e2a/R/loadReg2.R#L394-L397) or a data.frame with columns `fit`, `lwr`, and `upr` (https://github.com/wdwatkins/loadflex/blob/9e4e520e0b88a04ce594aa1e7b5048e26dae0e2a/R/loadReg2.R#L406-L420)